plugins/CoreAdminHome/lang/en.json

```diff
@@ -104,6 +104,10 @@
         "YouMayOptOutBis": "To make that choice, please click below to receive an opt-out cookie.",
         "OptingYouOut": "Opting you out, please wait...",
         "ProtocolNotDetectedCorrectly": "You are currently viewing Matomo over a secure SSL connection (using https), but Matomo could only detect a non secure connection on the server. ",
-        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s"
+        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s",
+        "MissingTrackingCodeEmailSubject": "%s is still missing the Matomo tracking code",
```

Comment:
I would maybe remove the word `still`. Or maybe we could say something like "No traffic for %s recorded yet, get started now".